app/models/education_benefits_claim.rb

```diff
@@ -2,10 +2,17 @@
 
 require 'attr_encrypted'
 class EducationBenefitsClaim < ApplicationRecord
-  FORM_TYPES = %w[1990 1995 1990e 5490 5495 1990n 0993 0994 1995s].freeze
-  FORM_HEADERS = [
-    '22-1990', '22-1995', '22-1990e', '22-5490', '22-5495', '22-1990n', '22-0993', '22-0994', '22-1995 STEM'
-  ].freeze
+  if Flipper.enabled?(:edu_benefits_stem_scholarship, @current_user)
```

Comment:
Are you enabling this for everyone, or just a percentage of users?  I don't think you need the @current_user parameter if you are enabling for all.
```suggestion
  if Flipper.enabled?(:edu_benefits_stem_scholarship)
```